apps/libsignpost/sara_u260.c

```diff
@@ -290,3 +290,60 @@ int sara_u260_get_post_partial_response(uint8_t* buf, size_t offset, size_t max_
 
     return dlen;
 }
+
+int sara_u260_get_ops_information(sara_u260_ops_info_t* inf, size_t num_info) {
+    int ret = at_send(SARA_CONSOLE,"AT+COPS=6\r"); 
+    if (ret < 0) return SARA_U260_ERROR;
+
+    //this size should allow us to receive numinfo neighbors
+    uint8_t* retbuf = malloc(num_info*50*sizeof(uint8_t));
+    if(!retbuf) {
+        return SARA_U260_ERROR;
+    }
+
+    int len = at_get_response(SARA_CONSOLE, num_info, retbuf, num_info*50);
```

Comment:
I think you should make a `retbuf_size` variable - your math is different here (no `sizeof`, which would technically be 1, but still) and it feels like duplicating the buffer size math is asking for trouble